deployment/refinery-modules/refinery/manifests/docker.pp

```diff
@@ -1,5 +1,19 @@
 class refinery::docker {
-  class { '::docker':
-    docker_users => [$::app_user]
+  $_docker_host = "unix:///var/run/docker.sock"
```

Comment:
The reason to set `DOCKER_HOST` on both AWS and in Vagrant is to enable communication using TCP in both cases. The only difference is that on AWS it's a separate box and in Vagrant it just happens to be localhost.